config/initializers/contacts.rb

```diff
@@ -0,0 +1,20 @@
+Rails.configuration.contacts = contacts = ActiveSupport::OrderedOptions.new
```

Comment:
This doesn't need to be in code. It can just be in copy. It's going to change very rarely and there's no need for it to be dynamic.